lib/utils.ts

```diff
@@ -23,6 +23,12 @@ export const isProductionStack = (stackId: string, rataExtraEnv: RataExtraEnviro
  */
 export const isFeatOrLocalStack = (rataExtraEnv: RataExtraEnvironment) =>
   rataExtraEnv === ENVIRONMENTS.feat || rataExtraEnv === ENVIRONMENTS.local;
```

Comment:
`isFeatOrLocalStack` to `isFeatStack`, no need to combine?

Reply:
We check for local, because all other environments are to be deploy to the cloud, and then the DB calls will fail because of the Prisma problem. This way we can use esbuild at least when doing LOCAL-stuff.